apps/web/pages/auth/login.tsx

```diff
@@ -1,7 +1,10 @@
 import { zodResolver } from "@hookform/resolvers/zod";
 import { jwtVerify } from "jose";
 import type { GetServerSidePropsContext } from "next";
-import { getCsrfToken, signIn } from "next-auth/react";
+import {
+  /*getCsrfToken*/
```

Comment:
This will impact actual deployed versions if we merge it in, right?Can we make it check for env in some way so we don't introduce a potential security issue in prod?

Reply:
its a var that is not being used

Reply:
Ohh I see there's a ton of commented out code here on the login page
Makes sense I guess :upside_down_face: